packages/office-ui-fabric-react/src/components/Modal/Modal.test.tsx

```diff
@@ -68,4 +69,87 @@ describe('Modal', () => {
 
     ReactDOM.createPortal.mockClear();
   });
+  it('renders a Modal with ARIA role alertDialog when isAlert is true', () => {
+    // Mock createPortal to capture its component hierarchy in snapshot output.
+    const ReactDOM = require('react-dom');
+    ReactDOM.createPortal = jest.fn(element => {
+      return element;
+    });
+
+    const component = renderer.create(
+      <Modal isOpen={true} isAlert={true} className={'test-className'} containerClassName={'test-containerClassName'}>
+        Test Content
+      </Modal>,
+    );
+
+    expect(component.root.findByType(Popup).props.role).toBe('alertdialog');
+    ReactDOM.createPortal.mockClear();
+  });
+  it('renders Modal with ARIA role dialog when isModeless and isBlocking are set to true', () => {
```

Comment:
nit: Add a line between these
```suggestion
  });
  
  it('renders Modal with ARIA role dialog when isModeless and isBlocking are set to true', () => {
```